### PR TITLE
Improve emit window

### PR DESCRIPTION
**Pull request by fernandohds564**

### CODE REVIEW

File: pyqt-apps/siriushla/as_ap_measure/emittance_meas_lnls.py [hunk 1]
```diff
@@ -144,6 +145,9 @@ def _acquire_data(self):
                     [min(sigma)*(1-DT)*1e3, max(sigma)*(1+DT)*1e3])
             self.plt_sigma.figure.canvas.draw()
         self._measuring.set()
+        _log.info('Returning Quad to Initial Current')
+        self.quad_I_sp.put(init_curr, wait=True)
```
Comment (murilobalves) [inline pyqt-apps/siriushla/as_ap_measure/emittance_meas_lnls.py]: great enhancement!

Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_measure/emittance_meas_lnls.py]: kkkk finally! :joy:

File: pyqt-apps/siriushla/as_ap_measure/emittance_meas_lnls.py [hunk 2]
```diff
@@ -507,3 +524,391 @@ def pb_stop_clicked(self):
...
+    y0 = y0 or np.mean(y)
+    try:
+        p_opt, p_cov = curve_fit(_gaussian, x, y, (amp, mu, sigma, y0))
+    except Exception:
+        p_opt = (amp, mu, sigma, y0)
+        print('Fitting Problem')
+    return p_opt
+
+
+class MatplotlibWidget(Canvas):
```
Comment (anacso17) [inline pyqt-apps/siriushla/as_ap_measure/emittance_meas_lnls.py]: This re-implementation could be moved to siriushla.widgets.figure_canvas.SiriusFigureCanvas.

Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_measure/emittance_meas_lnls.py]: done!

File: pyqt-apps/siriushla/as_ap_measure/emittance_meas.py [hunk 3]
```diff
@@ -404,8 +411,8 @@ def _setupUi(self):
         gl2.addWidget(QLabel('Vert.', resultsgb), 1, 2)
         gl2.addWidget(QLabel('Hor.', resultsgb), 1, 3)
         gl2.addWidget(QLabel('Vert.', resultsgb), 1, 4)
-        gl2.addWidget(QLabel('Norm. emitt. [mm.mrad]', resultsgb), 2, 0)
-        gl2.addWidget(QLabel('beta [m]', resultsgb), 3, 0)
+        gl2.addWidget(QLabel('Norm. emitt.\n[mm.mrad]', resultsgb), 2, 0)
```
Comment (fernandohds564) [inline pyqt-apps/siriushla/as_ap_measure/emittance_meas.py]: very nice @anacso17 !

File: pyqt-apps/siriushla/as_ap_measure/emittance_meas.py [hunk 4]
```diff
@@ -404,8 +411,8 @@ def _setupUi(self):
...
+        gl2.addWidget(QLabel('beta\n[m]', resultsgb), 3, 0)
```
Comment (fernandohds564) [inline pyqt-apps/siriushla/as_ap_measure/emittance_meas.py]: @anacso17, I don't think this line break is needed here, the whole beta label is already smaller than the first line of the emittance label. What do you think?

Comment (anacso17) [reply pyqt-apps/siriushla/as_ap_measure/emittance_meas.py]: @fernandohds564 my idea was to standardize, placing units in the second line. But I think it looks more pleasant without the line break, anyway!

Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_measure/emittance_meas.py]: Ok, I'll change that then, ok?

Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_measure/emittance_meas.py]: done!

**Pull request merged**